BedrockCore.h

```diff
@@ -5,6 +5,7 @@ class BedrockServer;
 
 class BedrockCore : public SQLiteCore {
   public:
+    const uint64_t DEFAULT_TIMEOUT = 60'000'000; // one minute.
```

Comment:
Thought about this later on... Why did you choose 1 minute? Seems really high.

Reply:
Because @righdforsa was afraid anything lower was risking catching too many commands, and this would still catch the worst cases. We can always adjust it later on, but I don't know if there's any other default that makes more sense without trying it like this.

Reply:
Well, have we looked at our longest running queries (I believe they're logged) and tried making a decision based on that existing data?

Reply:
We don't have this data for whole commands, only for queries. I just used logsearch to grab the last 2000 `Slow query` logs, which is just a little over an hour's worth (about 1h08m).

Of the 2,000 queries in that time period, 1877 (~94%) finished in under 10 seconds. Another 110 (~6%) finished in between 10 and 30 seconds. 10 (0.5%) queries finished in 30-60 seconds. Two queries took longer than a minute.

So, with the current settings, we're causing roughly 2 queries per hour to fail (assuming this hour of logs is representative), which is probably already more impact than we really want from just building this feature. If we changed it to 30 seconds, we'd be causing a query to fail roughly every 5 minutes due to timeout (note also, since these are individual query times, total commands will take longer, and be more likely to timeout than this).

60 seconds gives us protection against the most catastrophically slow commands, without much impacting our other commands. I'd be fine with lowering it further, if we first verified we rarely have any commands that take longer than the new limit, but I'm actually pretty OK with 60 seconds for now.

The two slow queries both seem to be `Get` with an RVL of `domain` for `keagan@expensify.com`.

Reply:
Sounds good.

>The two slow queries both seem to be Get with an RVL of domain for keagan@expensify.com.

Yikes 😕